KE-Apache-Beam-Analysis/README.md

```diff
@@ -0,0 +1,75 @@
+# Data Analytics @ Edge
+
+![High level architecture](Images/High_level_Arch.png "High Level Architecture")

```

Comment:
Can we have following structure in the README of each demo ?
Demo
| Description (detailed description of the demo)
| Prerequisites/SW & HW used (should specify kubeedge version used, RPi model, go version, etc)
| Steps to reproduce
| Additional Usecases (similar usecases if any)